addons/web/static/src/scss/popover.scss

```diff
@@ -52,12 +52,12 @@
     $arrow-pad: 8px;
     $arrow-size: 8px;
     $animation-time: 0.2s;
-    width: 280px;
```

Comment:
Not sure about this (but the fixed width seems to be a bad idea as well). Here, what prevents from having a super long single line popover?